src/sickle.ts

```diff
@@ -238,8 +252,8 @@ function last<T>(elems: T[]): T {
   return elems.length ? elems[elems.length - 1] : null;
 }
 
-export function annotate(file: ts.SourceFile): string {
-  return new Annotator().annotate(file);
+export function annotate(file: ts.SourceFile, options?: SickleOptions): string {
+  return new Annotator(options || defaultOptions).annotate(file);
```

Comment:
I'm not sure I followed you here -- what problem are you worried about, and what is your suggested fix?